test/Lower/character-assignement.f90

```diff
@@ -0,0 +1,103 @@
+! RUN: bbc %s -o "-" -emit-fir | FileCheck %s
```

Comment:
Probably want to rename this file to remove the `e` between between `assign` and `ment`.

Reply:
Thanks